### DISCUSSION TITLE
Open cryfs mode in Linux and in windows (dokany)

### DISCUSSION THREAD

**Discussion by realjench** [hardcore-sushi/DroidFS#178] [Q&A] [answered]
Hi

Is it possible to open the Cryfs mode in Linux ?

Or even the gocryptfs?
Thanks a lots :-)

**Comment by hardcore-sushi** [comment, marked as answer]
Yes absolutely, just use the original softwares:

- [cryfs](https://github.com/cryfs/cryfs)
- [gocryptfs](https://github.com/rfjakob/gocryptfs)

**Reply by realjench** [reply]
Thanks but

When I create a cryfs "container" the config file is totally unreadable.

And with DroidFs it does look like this:

`{
	"Creator": "DroidFS",
	"EncryptedKey": "D0ewF7hgr4sea5xa6sjIu1zy3U0DKUWZY7jknnpw6vZTZzLXzWsmKIE46MkQYaVtl48JUmI19MeFClaC8jt2FA==",
	"ScryptObject": {
		"Salt": "x7S/xAAvNQuWSrzRewaPTEnAqDCfc1501jtnpQkIbx4=",
		"N": 65536,
		"R": 8,
		"P": 1,
		"KeyLen": 32
	},
	"Version": 2,
	"FeatureFlags": [
		"HKDF",
		"XChaCha20Poly1305",
		"DirIV",
		"EMENames",
		"LongNames",
		"Raw64"
	]
}`


How may it work?

How make Cryfs accept a json configuration file?


Thanks a lot, I must be a bit dumb, sorry

**Reply by realjench** [reply]
By the way, I created a cryfs from linux, tried to open it in Droidfs, but he tells me, that's not a gocryptfs not reconized

**Reply by hardcore-sushi** [reply]
OK, it looks like you're using the 1.x.x version of DroidFS, which doesn't support CryFS. If you want CryFS support, you have to upgrade to the 2.x.x alpha version of DroidFS (not available on F-Droid yet). Then, when creating a volume, select "CryFS" as volume type.

**Reply by realjench** [reply]
Thanks a lot for your answer.

I forgot to mention the version used, sorry to make you lost some times.

I'll check DroidFS 2x Alpha for non sensitive dataz.

Have a great day

**Reply by hardcore-sushi** [reply]
No problem, feel free to ask if you have any other questions.
